lnwire/lnwire_test.go

```diff
@@ -603,7 +603,7 @@ func TestLightningWireProtocol(t *testing.T) {
 			req := ChannelUpdate{
 				ShortChannelID:  NewShortChanIDFromInt(uint64(r.Int63())),
 				Timestamp:       uint32(r.Int31()),
-				Flags:           ChanUpdateFlag(r.Int31()),
+				ChannelFlags:    ChanUpdateFlag(r.Int31()),
```

Comment:
also set `MsgFlags`? Same question goes several places in this commit.

Reply:
fixed